ERROR - 2022-10-08 12:34:56 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 12:34:56 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 12:38:29 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 12:38:29 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 12:39:58 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 12:39:58 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 12:42:11 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 12:42:11 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 12:43:08 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 12:43:08 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 12:59:27 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 12:59:27 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:04:58 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:04:58 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:06:13 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:06:13 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:06:33 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:06:33 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:07:28 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:07:28 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:07:29 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:07:29 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:07:30 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:07:30 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:08:22 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:08:22 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:08:37 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:08:37 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:09:13 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:09:13 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:09:14 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:09:14 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:09:15 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:09:15 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
ERROR - 2022-10-08 13:09:15 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-10-08 13:09:15 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\backend-delivery\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\backend-delivery\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\backend-delivery\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\backend-delivery\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(32): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->login()
#6 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#7 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2022-10-08 15:58:05 --> Undefined array key 1
#0 C:\laragon\www\backend-delivery\app\Controllers\AuthController.php(82): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\laragon\\www\\...', 82)
#1 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->getUserData()
#2 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#3 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#6 {main}
CRITICAL - 2022-10-08 16:29:06 --> Cannot access protected property CodeIgniter\HTTP\IncomingRequest::$body
#0 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->getUserData()
#1 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#2 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#5 {main}
CRITICAL - 2022-10-08 18:07:56 --> Call to a member function getValue() on null
#0 C:\laragon\www\backend-delivery\system\CodeIgniter.php(825): App\Controllers\AuthController->getUserData()
#1 C:\laragon\www\backend-delivery\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#2 C:\laragon\www\backend-delivery\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\laragon\www\backend-delivery\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\laragon\www\backend-delivery\system\Commands\Server\rewrite.php(43): require_once('C:\\laragon\\www\\...')
#5 {main}
ERROR - 2022-10-08 23:27:06 --> Cannot delete or update a parent row: a foreign key constraint fails (`deliverybd`.`tienda`, CONSTRAINT `fk_tienda_categoriaTienda` FOREIGN KEY (`id_categoria`) REFERENCES `categoria_tienda` (`id`))
ERROR - 2022-10-08 23:28:43 --> Cannot delete or update a parent row: a foreign key constraint fails (`deliverybd`.`tienda`, CONSTRAINT `fk_tienda_categoriaTienda` FOREIGN KEY (`id_categoria`) REFERENCES `categoria_tienda` (`id`))
